charts/opentelemetry-operator/values.yaml

```diff
@@ -32,7 +32,7 @@ manager:
     tag: ""
   collectorImage:
     repository: otel/opentelemetry-collector-contrib
-    tag: 0.93.0
+    tag: 0.94.0
```

Comment:
can we bump this image to [v.0.95.0 ](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.95.0)?